internal/clientconn/listener.go

```diff
@@ -53,6 +55,9 @@ type NewListenerOpts struct {
 	TestRecordPath     string // if empty, no records are created
 }
 
+// Default IP we are listening on
+const defaultListeningIP = "127.0.0.1:27017"
```

Comment:
Let's not do that; let's compare with an empty string below. If someone wants to disable TCP listener, they should pass `--listen-addr=""`

Reply:
I agree that's it's certainly a code smell.
I was just thinking I someone passes only `--listen-sock` then it must be only sock.

But'll fix it.